extern/storage-fsm/precommit_policy_test.go

```diff
@@ -29,7 +37,7 @@ func TestBasicPolicyEmptySector(t *testing.T) {
 	exp, err := policy.Expiration(context.Background())
 	require.NoError(t, err)
 
-	assert.Equal(t, 3455, int(exp))
+	assert.Equal(t, 2879, int(exp))
```

Comment:
All of these constants (this and below) needed to be changed to get the tests to pass. Is this correct?

Reply:
I'm going to guess that we used very old actors in storage-fsm, this is probably fine